src/app/containers/IndexHeading/index.stories.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { latin } from '@bbc/gel-foundations/scripts';
+import IndexHeading from '.';
+
+storiesOf('Containers|Index Heading', module)
+  .addParameters({ chromatic: { disable: true } })
+  .add('default', () => (
+    <IndexHeading script={latin} service="news">
```

Comment:
Can we have the examples map to actual IDX examples? e.g.
```suggestion
    <IndexHeading script={cyrillicAndLatin} service="ukrainian">
```

Reply:
Sure, needed to update those :)